packages/vanilla/src/sass/components/checkbox/_index.scss

```diff
@@ -2,6 +2,7 @@
   border: 0;
   padding: 0;
   margin: 0;
+  margin-bottom: var(--cbp-space-4x);
```

Comment:
I know you guys may be on the fence, but Jeremy said he was ok with a default that can be overridden when we discussed this before. I've been round and round on this before and seen it in use though. The main context for input patterns is stacking them within forms. Not having a default spacing below each input patter will cause developers to do all sorts of stuff like put forms in a flex column to provide this spacing (which has side-effects similar to one of the comments above). It's much better to provide this default for the form context and allow it to be overridden when needed (often removing it, which is the simplest because 0 doesn't require thought).|

We could key this to a descendant of a form tag if you like but there are very few use cases where it makes sense to have inputs outside of a form, so I'm not sure if that solves much.